packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.tsx

```diff
@@ -2,14 +2,11 @@ import React, { useEffect, useContext } from 'react';
 import { Plan } from '../../../store/types';
 import { AppContext } from '../../../lib/AppContext';
 
+import fpnImage from '../../../images/fpn';
```

Comment:
I decided to import this since as of this PR, we're using this image in 3 places.

I created a separate issue to address product image scalability and making this less painful: https://github.com/mozilla/fxa/issues/3039